templates/themes/horizon/invenio_communities/details/home/index.html

```diff
@@ -48,19 +48,76 @@ <h1 class="ui large header">{{ _("Open repository for EU-funded research (Pilot)
         </div>
       </div>
     </div>
+
+  <!--BROWSE SECTION-->
+  {% macro format_metric_value(title, value) %}
+    {% if title == "total_records" or title == "total_grants" %}
+      {% if value >= 1000000 %}
+        {{ (value // 1000000) }} M
+      {% elif value >= 100000 %}
+        {{ '{:,.0f}'.format(value // 1000) }} K
+      {% else %}
+        {{ '{:,.0f}'.format(value) }}
+      {% endif %}
+    {% elif title == "total_data" %}
+      {% if value >= 1000 %}
+        {{ (value / 1000) | round(1) }} TB
+      {% else %}
+        {{ value }} GB
+      {% endif %}
+    {% else %}
+      {{ value }}
+    {% endif %}
+  {% endmacro %}
```

Comment:
minor/shelve: this should be definitely a Jinja filter in Python, I can see it wasn't fun to write 😅

Reply:
Actually for the formatting, why not use the existing functions we use for views/downloads: [`filesizeformat`](https://github.com/slint/invenio-app-rdm/blob/a96d89a4514e10a82b9f1a09b21d3a6d44515520/invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/stats.html#L113) and [`compact_number`](https://github.com/slint/invenio-app-rdm/blob/a96d89a4514e10a82b9f1a09b21d3a6d44515520/invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/stats.html#L14)